lib/oprt.gd

```diff
@@ -1587,12 +1626,21 @@ OrbitsishOperation( "ExternalOrbitsStabilizers", OrbitsishReq,
 ##  <P/>
 ##  An action is <E><M>k</M>-transitive</E> if every <M>k</M>-tuple of points
 ##  can be mapped simultaneously to every other <M>k</M>-tuple.
+##  <P/>
+##  For a permutation group <A>G</A>, one may also invoke this as
+##  <C>Transitivity(<A>G</A>)</C>, which returns the degree of transitivity
+##  of the group with respect to its natural action on the points moved by
+##  it.
+##  For example the group <M>\langle (2,3,4),(2,3) \rangle</M>
+##  is 3-transitive on the three points 2, 3, 4.
```

Comment:
Technically, on usually defines a group to act 3-fold transitive on a set, not on points. So maybe:
_is 3-transitive on the set {2,3,4}._?

Reply:
I changed it now in all bits, but it now feels inconsistent with the sentence before.